docs/setup/storage/storage.md

```diff
@@ -454,6 +454,16 @@ And you should configure oss related properties in file **zeppelin-site.xml**.
   <description>Access key secret for your OSS account</description>
 </property>
 
+<property>
+  <name>zeppelin.notebook.oss.version.max</name>
+  <value>30</value>
```

Comment:
It's just an opinion but how about disabling this feature at first? Otherwise, if we set it to 30 in a non-history folder structure, it would be handled automatically?

Reply:
Hi @jongyoul , It's ok if we set it to 30 in a non-history folder structure.  Actually non-history folder is a just a special case of history folder because I have handled that in the code.  The new OSSNotebookRepo is completely compatible with this situation